src/api/public/apidocs-new/paths/build.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  summary: Get a simple directory listing of all projects
+  description: Get a simple directory listing of all projects
+  security:
+    - basic_authentication: []
+  responses:
+    '200':
+      description: |
+        OK
+
+        XML Schema used for body validation: [directory.xsd](../schema/directory.xsd)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/directory.yaml'
+          example:
+            entry:
+              - name: 'openSUSE:Leap:15.1'
+              - name: 'openSUSE:Leap:15.2'
```

Comment:
This response 200 returning a list of projects is very common. Does it make sense to move it to its own file and add a reference to it whenever it's needed?

Reply:
Makes sense, as discussed lets do it in a batch as a follow up :)